packages/e2e/test/wallet/PersonalWallet/conwayTransactions.test.ts

```diff
@@ -0,0 +1,228 @@
+  };
+
+  const getPoolId = async () => {
+    const activePools = await wallet.stakePoolProvider.queryStakePools({
+      filters: { status: [Cardano.StakePoolStatus.Active] },
+      pagination: { limit: 1, startAt: 0 }
+    });
+
+    poolId = activePools.pageResults[0].id;
+  };
+
+  const getStakeCredential = async () => {
+    const rewardAcct = await firstValueFrom(wallet.addresses$.pipe(map((addresses) => addresses[0].rewardAccount)));
+
+    stakeCredential = {
+      hash: Crypto.Hash28ByteBase16.fromEd25519KeyHashHex(Cardano.RewardAccount.toHash(rewardAcct)),
+      type: Cardano.CredentialType.KeyHash
+    };
+  };
+
+  const feedDRepWallet = async () => {
+    const balance = await firstValueFrom(dRepWallet.balance.utxo.total$);
+
+    if (balance.coins > 10_000_000n) return;
+
+    const address = await firstValueFrom(dRepWallet.addresses$.pipe(map((addresses) => addresses[0].address)));
+
+    const signedTx = await wallet
+      .createTxBuilder()
+      .addOutput({ address, value: { coins: 20_000_000n } })
+      .build()
+      .sign();
+
+    await submitAndConfirm(wallet, signedTx.tx, 1);
+  };
+
+  beforeAll(async () => {
+    [wallet, dRepWallet] = await Promise.all([
+      getTestWallet(0, 'Conway Wallet', 100_000_000n),
+      getTestWallet(1, 'Conway DRep Wallet', 0n)
+    ]);
+
+    await Promise.all([getDeposits(), getPoolId(), getStakeCredential(), feedDRepWallet()]);
+  });
+
+  beforeEach(cleanWallet);
+
+  afterAll(async () => {
+    await cleanWallet();
+    wallet.shutdown();
+    dRepWallet.shutdown();
+  });
+
+  // We can't do this yet. Fee is not calculated because this certificate's deposit is not take into account by input selection
+  it.skip('can send a dRep registration certificate and find it on chain', async () => {
+    const txBuilder = wallet.createTxBuilder();
+    const drepPubKey = await wallet.getPubDRepKey();
+    const dRepKeyHash = Crypto.Hash28ByteBase16.fromEd25519KeyHashHex(
+      (await Crypto.Ed25519PublicKey.fromHex(drepPubKey).hash()).hex()
+    );
+    const dRepRegCert: Cardano.RegisterDelegateRepresentativeCertificate = {
+      __typename: Cardano.CertificateType.RegisterDelegateRepresentative,
+      anchor,
+      dRepCredential: { hash: dRepKeyHash, type: Cardano.CredentialType.KeyHash },
+      deposit: 0n
+    };
+
+    txBuilder.customize(({ txBody }) => ({ ...txBody, certificates: [...(txBody.certificates || []), dRepRegCert] }));
+    const signedTx = await txBuilder.build().sign();
+    await wallet.submitTx(signedTx);
+
+    // Wait for the transaction to show up in chain history
+    const txFoundInHistory = await firstValueFromTimed(
+      wallet.transactions.history$.pipe(
+        map((txs) => txs.find((tx) => tx.id === signedTx.tx.id)),
+        filter(isNotNil),
+        take(1)
+      )
+    );
+
+    expect(txFoundInHistory.body.certificates?.length).toBe(1);
+    expect(txFoundInHistory.body.certificates![0]).toEqual(dRepRegCert);
```

Comment:
```diff
-    expect(txFoundInHistory.body.certificates![0]).toEqual(dRepRegCert);
+     assertTxHasCertificate...

Reply:
I've not worked yet on this test... it needs to be completely reworked